lib/stat_tracker.rb

```diff
@@ -1,2 +1,97 @@
+require 'CSV'
+require './lib/team'
+require './lib/game'
+
 class StatTracker
+
+  def initialize
+    @teams = []
```

Comment:
This will instantiate all of our objects one time, and our methods will pull from this.